aarch64 seems to run into a spurious test failure:

```
==================================== ERRORS ====================================
_______________ ERROR at setup of test_remove_older_logs[pyloop] _______________
[gw6] linux -- Python 3.8.6 /nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/bin/python3.8
loop_factory = <class 'asyncio.unix_events._UnixDefaultEventLoopPolicy'>
fast = False, loop_debug = False
    @pytest.fixture
    def loop(loop_factory, fast, loop_debug):  # type: ignore
        """Return an instance of the event loop."""
        policy = loop_factory()
        asyncio.set_event_loop_policy(policy)
>       with loop_context(fast=fast) as _loop:
/nix/store/6ij587xhxfxngkx8d0brwgbhxv2ai1pj-python3.8-aiohttp-3.7.3/lib/python3.8/site-packages/aiohttp/pytest_plugin.py:228:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/contextlib.py:113: in __enter__
    return next(self.gen)
/nix/store/6ij587xhxfxngkx8d0brwgbhxv2ai1pj-python3.8-aiohttp-3.7.3/lib/python3.8/site-packages/aiohttp/test_utils.py:493: in loop_context
    loop = setup_test_loop(loop_factory)
/nix/store/6ij587xhxfxngkx8d0brwgbhxv2ai1pj-python3.8-aiohttp-3.7.3/lib/python3.8/site-packages/aiohttp/test_utils.py:514: in setup_test_loop
    asyncio.set_event_loop(loop)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/events.py:753: in set_event_loop
    get_event_loop_policy().set_event_loop(loop)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/unix_events.py:1365: in set_event_loop
    self._watcher.attach_loop(loop)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/unix_events.py:931: in attach_loop
    self._do_waitpid_all()
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/unix_events.py:985: in _do_waitpid_all
    self._do_waitpid(pid)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/unix_events.py:1019: in _do_waitpid
    callback(pid, returncode, *args)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/unix_events.py:216: in _child_watcher_callback
    self.call_soon_threadsafe(transp._process_exited, returncode)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/base_events.py:764: in call_soon_threadsafe
    self._check_closed()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <_UnixSelectorEventLoop running=False closed=True debug=False>
    def _check_closed(self):
        if self._closed:
>           raise RuntimeError('Event loop is closed')
E           RuntimeError: Event loop is closed
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/asyncio/base_events.py:508: RuntimeError
```

--- a/pkgs/servers/home-assistant/component-packages.nix
+++ b/pkgs/servers/home-assistant/component-packages.nix
@@ -2,7 +2,7 @@
 # Do not edit!
 
 {
-  version = "2021.1.3";
+  version = "2021.1.4";
   components = {
     "abode" = ps: with ps; [ abodepy ];
     "accuweather" = ps: with ps; [ accuweather ];

--- a/pkgs/servers/home-assistant/default.nix
+++ b/pkgs/servers/home-assistant/default.nix
@@ -57,7 +57,7 @@ let
   extraBuildInputs = extraPackages py.pkgs;
 
   # Don't forget to run parse-requirements.py after updating
-  hassVersion = "2021.1.3";
+  hassVersion = "2021.1.4";
 
 in with py.pkgs; buildPythonApplication rec {
   pname = "homeassistant";
@@ -76,7 +76,7 @@ in with py.pkgs; buildPythonApplication rec {
     owner = "home-assistant";
     repo = "core";
     rev = version;
-    sha256 = "168njrzvqr3wn1xk02q1yn1f90mf1hds6m8lffgkql8m6a6576hn";
+    sha256 = "03aa7kd216rnp8h80nv002ahafiy0031lxk1bkwcirrznphcw7sj";
   };
 
   # leave this in, so users don't have to constantly update their downstream patch handling
@@ -179,6 +179,8 @@ in with py.pkgs; buildPythonApplication rec {
     # keyring.errors.NoKeyringError: No recommended backend was available.
     "test_secrets_from_unrelated_fails"
     "test_secrets_credstash"
+    # RuntimeError: Event loop is closed
+    "test_remove_older_logs"
   ];
 
   preCheck = ''

